### ISSUE TITLE
Special tags of `set_header_footer()` give me a corrupt file

### ISSUE THREAD

**Issue by olivroy** [JanMarvin/openxlsx2#747]
Running the examples of `wb_set_header_footer()` give me a corrupt file when using the special tags.

I am running Microsoft Excel 365 on Windows 10, with R 4.2.3 in a French locale..

```r
wb <- wb_workbook()

# Add example data
wb$add_worksheet("S1")$add_data(x = 1:400)
wb$set_header_footer(
sheet = 1,
header = c("&[Date]", "ALL HEAD CENTER 2", "&[Page] / &[Pages]"),
footer = c("&[Path]&[File]", NA, "&[Tab]"),
first_header = c(NA, "Center Header of First Page", NA),
first_footer = c(NA, "Center Footer of First Page", NA)
)
wb$open()
```

If you can't reproduce, I will provide more details!

It's written Illegal character name

Edit: I encounter the same `wb$add_worksheet()`  and `wb$set_header_footer()`

**Comment by JanMarvin** [comment]
Thanks for the report! A fix is in #748

**Comment by olivroy** [comment]
Awesome thanks!
